backend/src/main/java/com/example/backend/service/EquipmentItemService.java

```diff
@@ -47,4 +47,16 @@ public EquipmentItem updateEquipmentItem(EquipmentItemDTO equipmentItemDTO, Stri
                 .done(equipmentItemDTO.isDone())
                 .build());
     }
+
+    public String deleteEquipmentItem(String id) {
+        if (equipmentItemRepository.findById(id).isPresent()) {
+            equipmentItemRepository.deleteById(id);
+            if (equipmentItemRepository.findById(id).isEmpty()) {
+                return id;
+            } else {
+                return "Deletion failed";
+            }
```

Comment:
I like that you test if the deletion really happened. Nice. My 2 cents:
```suggestion
            return equipmentItemRepository.findById(id).isEmpty() ? id : "Deletion failed"; 
```

Reply:
Good hint, but I stay with "old" if else structure. Why?
I don't like to mix the Syntax, so I would have to use shorthand for both ifs. When doing so, it will be less easy to read for me, allthough it would spare lines of code